odl/tomo/backends/stir_bindings.py

```diff
@@ -170,7 +168,7 @@ def adjoint(self):
 
 class BackProjectorByBinWrapper(Operator):
 
-    """A backprojector using STIR."""
+    """A back projector using STIR."""
```

Comment:
I'm never sure which one is right, you see both often. Shall we agree on "back projector"? Let's be consistent then (also in class names!).